editor/src/components/text-editor/text-editor.spec.browser2.tsx

```diff
@@ -209,6 +209,41 @@ describe('Use the text editor', () => {
       expect(after).toEqual(projectWithStyle('textDecoration', 'none'))
     })
 
+    it('supports increasing font size', async () => {
+      const { before, after } = await testModifier(
+        { shift: true, cmd: true, alt: false, ctrl: false },
+        '.',
+      )
+      expect(before).toEqual(projectWithStyle('fontSize', '17px'))
+      expect(after).toEqual(projectWithStyle('fontSize', '18px'))
+    })
+
+    it('supports increasing font weight', async () => {
+      const { before, after } = await testModifier(
+        { shift: false, cmd: true, alt: true, ctrl: false },
```

Comment:
worth adding an exported `altCmdModifier` or `optCmdModifier` and use it here

Reply:
done!